src/storage/file_io.h

```diff
@@ -180,6 +180,11 @@ struct fileio_page_reserved
   INT32 p_reserve_1;
   INT32 p_reserve_2;		/* unused - Reserved field */
   INT64 tde_nonce;		/* tde nonce. atomic counter for temp pages, lsa for perm pages */
+
+  inline bool operator== (const fileio_page_reserved & other) const
```

Comment:
This operator does not seem to be complete.
If it is needed to compare two `fileio_page_reserved` incompletely in a specific context, better implement an out-of-type comparison function than pollute the type definition with an incomplete implementation.

Reply:
This is what Remzi considered enough.